tx-metadata.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
```

Comment:
Could you rename this file as `test-metadata.js` so it's consistent with the other test files?